jobs/env/ci-kubernetes-e2e-scalability.env

```diff
@@ -1,19 +1,19 @@
 ### job-env
```

Comment:
Maybe rename this file to `ci-kubernetes-e2e-gce-scalability-common.env` for more clarity?

Reply:
Done